### PR TITLE
download_older_versions?

### PR DESCRIPTION
**Pull request by Etherum1914**
Requesting ability to download older Checkra1n versions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/free-form.md [hunk 1]
```diff
@@ -1,16 +1,2 @@
...
----
-
-<!--
-
-DO NOT USE THIS TEMPLATE FOR BUG REPORTS!
-
-IF YOU DO, THEY WILL BE CLOSED AND MARKED AS INVALID!
-
--->
+## Ability to download older versions of Checkra1n?
```
Comment (gabriel-1988) [inline .github/ISSUE_TEMPLATE/free-form.md:1]: Hey

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:1]: Yoga

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:1]: Ghh y kg

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:1]: B Ku de

File: .github/ISSUE_TEMPLATE/free-form.md [hunk 2]
```diff
@@ -1,16 +1,2 @@
...
+It would be very nice to have the ability to download older verions of Checkra1n in case newer versions happen to not work so well. For me, version 0.10.2 does not work 99% of the time. It gets stuck between steps and does not progress. No help has come from research or asking for help. No, I'm not reporting a bug. I'm asking for the ability for us to get older versions and not be forced to use the newest. I was successful with version 0.9.x.x but I am unable to get it as it's no longer available.
```
Comment (gabriel-1988) [inline .github/ISSUE_TEMPLATE/free-form.md:2]: Gif

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: Bkhk

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: Tj

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: Hjhh

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: - Vjghg

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: GBox

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: Vhgj

Comment (gabriel-1988) [reply .github/ISSUE_TEMPLATE/free-form.md:2]: Chih

**Review by gabriel-1988 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/free-form.md (hunk 1 above)
Comment (Etherum1914) [inline .github/ISSUE_TEMPLATE/free-form.md:1]: Are you having a stroke? Plus I found a solution about this.

Comment (Etherum1914) [reply .github/ISSUE_TEMPLATE/free-form.md:1]: I found where the older versions are. You guys can ignore this one and I do apologize.

**Pull request closed without merging**